Sources/main.swift

```diff
@@ -1,18 +1,22 @@
 import Foundation
 import SocketIO
 
-let socket = SocketIOClient(socketURL: URL(string: "http://localhost:8080/")!, config: [.log(false)])
+let manager = SocketManager(socketURL: URL(string: "http://localhost:8080")!, config: [.log(true), .compress])
+let socket = manager.defaultSocket
 
-socket.on("connect") {data, ack in
-    print("socket connected")
-    print("Type \"quit\" to stop")
+socket.on("connect") { data, ack in
+    print("クライアントとつながりました")
```

Comment:
Would be nice to keep some english

Reply:
I appreciate your kind advice☺️.
I try to change this pull request to good pull request!